tests/CurrentConversation.spec.ts

```diff
@@ -0,0 +1,7 @@
+import { test, expect } from '@playwright/test';
+
+test('test', async ({ page }) => {
+  await page.goto('http://localhost:3000/conversations');
+  await page.getByTestId('item-donor').click();
+  expect(await page.waitForSelector('p[data-testid="item-donor"]'));
+});
```

Comment:
Nice one. But I think we need to fall on some conventions with test suites because as they grow it will become a nightmare to read through these logs.

```js
test.describe('CurrentConversation', ()=>{ // the component you are testing
  test('Displays information on Conversation Partner', async ({ page }) => { // the exact action your test checks
   ... your test
  });
})
```

Reply:
Agree with this! Will update now

Reply:
I think we could refactor this test to pick up on the problem I'm finding on my branch.

If you want I can pair up with on re-writing the test before you try to fix that bug